src/Grpc.Net.Client/Balancer/SubchannelsLoadBalancer.cs

```diff
@@ -122,6 +122,7 @@ public override void UpdateChannelState(ChannelState state)
 
         var allUpdatedSubchannels = new List<AddressSubchannel>();
         var newSubchannels = new List<Subchannel>();
+        var modifiedSubchannelsCount = 0;
```

Comment:
It doesn't seem important either way, but it looks like this could be a simple bool.